cohort-engine-api-web/src/test/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandlerTest.java

```diff
@@ -120,88 +141,144 @@ public void setUp() throws Exception {
 	}
 
 	@BeforeClass
-	public static void setUpBeforeClass() throws Exception {
+	public static void setUpHandlerBeforeClass() throws Exception {
 		restHandler = new CohortEngineRestHandler();
 	}
 
 	/**
 	 * Test the successful building of a response.
 	 */
-
 	@PrepareForTest({ Response.class, TenantManager.class, ServiceBaseUtility.class })
 	@Test
-	public void testEvaluateMeasuresStub() throws Exception {
+	public void testEvaluateMeasureSuccess() throws Exception {
 		prepMocks();
+		
 		PowerMockito.mockStatic(ServiceBaseUtility.class);
-		PowerMockito.mockStatic(Response.class);
-		PowerMockito.mockStatic(ResponseBuilder.class);
 		PowerMockito.when(ServiceBaseUtility.apiSetup(version, logger, methodName)).thenReturn(null);
-		PowerMockito.when(Response.status(Response.Status.ACCEPTED)).thenReturn(mockResponseBuilder);
-		PowerMockito.when(mockResponseBuilder.entity(Mockito.anyString())).thenReturn(mockResponseBuilder);
-		PowerMockito.when(mockResponseBuilder.header(Mockito.anyString(), Mockito.anyString()))
-				.thenReturn(mockResponseBuilder);
-		when(mockResponseBuilder.build()).thenReturn(mockResponse);
-		Response loadResponse = restHandler.evaluateMeasures(mockRequestContext, "version", new MeasuresEvaluation());
+		
+		mockResponseClasses();
```

Comment:
Wiremock creates an actual web server during test initialization that will respond to requests with known responses. I use it as a pretend FHIR server and you can see it layered into the tests I wrote here. I'm not sure, though, that it is a complete replacement for the mocking around HTTP requests and responses that are in these tests. I would be willing to explore it more, but that would be in a separate ticket. The mockResponseClasses() method here was just a refactoring to avoid restating the same setup code in multiple tests.